dataverse-persistence/src/main/resources/Bundle_en.properties

```diff
@@ -2734,6 +2734,9 @@ ingest.error.EXCEL_ONLY_ONE_ROW=Only one row of data (column name header?) detec
 ingest.error.EXCEL_READ_FAIL=Failed to read line {0} during the second pass.
 ingest.error.EXCEL_MISMATCH=Reading mismatch, line {0} during the second pass: {1} delimited values expected, {2} found.
 ingest.error.EXCEL_LINE_COUNT=Mismatch between line counts in first and final passes!
+ingest.error.EXCEL_AMBIGUOUS_INDEX_POSITION=Could not establish position index of a cell element unambiguously.
+ingest.error.EXCEL_UNKNOWN_VARIABLE_NAME=Could not establish variable name for column {0}.
+ingest.error.EXCEL_UNKNOWN_OR_INVALID_COLUMN_COUNT=Could not establish column count, or invalid column count encountered.
```

Comment:
We also want to modify `ingest.error.EXCEL_PARSE` - so it would not contain any argument.
`Could not parse Excel/XLSX spreadsheet. {0}` -> `Could not parse Excel/XLSX spreadsheet.` (when we throw `IngestException(IngestError.EXCEL_PARSE)` we do not include any param)